fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseData.java

```diff
@@ -88,7 +87,8 @@ public Response(
             SimplifiedIntervalList volatileIntervals,
             Map<String, URI> paginationLinks,
             Pagination pagination,
-            ObjectMappersSuite objectMappers
+            ObjectMappersSuite objectMappers,
+            ApiRequest apiRequest
```

Comment:
We can probably also drop the `ResponseFormatType` parameter, since the writers are pulling that off the `ApiRequest`.